### PR TITLE
Change the field escuela in the Integrante form to plantel. And add a

### PR DESCRIPTION
**Pull request by alexgamaes**
optional value escuela to all members

#### What's this PR do?
Change the field escuela in the Integrante form to plantel. And add an optional value escuela to all members

#### Where should the reviewer start?
First, he would to check the file familias/models.py, and see that a the new field escuela was added to Integrante. 
Next, view familias/form and review the changes to IntegranteModelForm, on which escuela was changed to plantel.
Finally, you can also view captura/views.py, in which I change escuela to plantel, and the tests.

#### How should this be manually tested?
Enter as a Capturista to a socioeconomical study, and create a new integrante. Write something in the school field, and if the integrante is an alumno, also you must to choose a plantel. 

After you save, edit the integrante to see that also update works.

#### Any background context you want to provide?
This has been added as per request by stakeholders.

This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)

### CODE REVIEW

File: familias/models.py [hunk 1]
```diff
@@ -54,6 +54,9 @@ class Familia(models.Model):
     estado_civil : TextField
         This field stores the information regarding the legal relationship status of the
         parents in a family.
+    escuela : TextField
```
Comment (AndreeToledo) [inline familias/models.py]: Please change this field, to a Charfield, so it renders nicely in the form

File: familias/models.py [hunk 2]
```diff
@@ -222,6 +225,7 @@ class Integrante(models.Model):
                                              verbose_name=sacramentos)
     hist = '¿Asiste o asistió a terapia por alguna situación de AA, psicología, psiquiatría, etc.?'
     historial_terapia = models.TextField(blank=True, verbose_name=hist)
+    escuela = models.TextField(max_length=200, blank=True)
```
Comment (AndreeToledo) [inline familias/models.py]: Please change this to a CharField so it displays nicely on the form for integrante

**Comment by AndreeToledo** [comment]
Y porfavor checa las migraciones que tienen conflicto con las que están en develop

**Comment by alexgamaes** [comment]
Listo!

**Comment by AndreeToledo** [comment]
Conflicting migrations, are still there, please make a manual merge from develop to your branch, once that's done check the migrations and fix the conflicts.

**Pull request merged**